modules/javafx.graphics/src/main/native-iio/jpegloader.c

```diff
@@ -1338,6 +1341,8 @@ JNIEXPORT jlong JNICALL Java_com_sun_javafx_iio_jpeg_JPEGImageLoader_initDecompr
         char buffer[JMSG_LENGTH_MAX];
         (*cinfo->err->format_message) ((struct jpeg_common_struct *) cinfo,
                 buffer);
+        free(cinfo->err);
+        free(cinfo);
```

Comment:
jerr_mgr is also allocated via malloc, but not freed. Do you want to free that too?

Reply:
Line: 1332=>   jerr_mgr ptr is stored in cinfor->err
`cinfo->err = jpeg_std_error(&(jerr_mgr->pub));`

so free(cinfo->err) is same as free (jerr_mgr).

free(cinfo->err) is used here [instead of free(jerr_mgr)] to match the free call in imageio_dispose() method.